src/lib_rules.ml

```diff
@@ -374,6 +374,7 @@ module Gen (P : Install_rules.Params) = struct
         ~lib_name:(Some lib.name)
         ~dir_kind
     in
+    Ocamlformat.gen_rules sctx lib.buildable ~dir ~scope modules;
```

Comment:
Is this used for ocaml files only or also for reasonML files ?

Reply:
now, this ignores reason files